src/main/webapp/review.js

```diff
@@ -116,9 +116,8 @@ function storeComment(email) {
   params.append('rating', rating.options[rating.selectedIndex].value)
   params.append('content', content);
   params.append('store', localStorage.getItem("store"));
-  console.log(store);
-  console.log("store hit");
   params.append('email', email);
+  window.location.reload();
```

Comment:
Since we're reloading the page, line 125-132 (originally line 124-131) would never be used and we can just delete it.

Reply:
fixed this location, didn't notice it wasn't at the end of a function